test/spec/modules/greenbidsAnalyticsAdapter_spec.js

```diff
@@ -149,48 +150,51 @@ describe('Greenbids Prebid AnalyticsAdapter Testing', function () {
         });
       }
 
-      describe('#createCommonMessage', function() {
-        it('should correctly serialize some common fields', function() {
+      describe('#createCommonMessage', function () {
+        it('should correctly serialize some common fields', function () {
           const message = greenbidsAnalyticsAdapter.createCommonMessage(auctionId);
 
           assertHavingRequiredMessageFields(message);
         });
       });
 
-      describe('#serializeBidResponse', function() {
-        it('should handle BID properly with timeout false and hasBid true', function() {
+      describe('#serializeBidResponse', function () {
+        it('should handle BID properly with timeout false and hasBid true', function () {
           const result = greenbidsAnalyticsAdapter.serializeBidResponse(receivedBids[0], BIDDER_STATUS.BID);
 
           expect(result).to.include({
             bidder: 'greenbids',
             isTimeout: false,
             hasBid: true,
           });
+          expect(result.params).to.eql({});
```

Comment:
todo: Je ne suis pas d'accord. Ce test ci vérifie "should handle BID properly with timeout false and hasBid true", il n'a que faire de `params` dans tu ne devrais faire aucune assertion dessus.